-------------------------------------------------------------------------------
Test set: org.apache.hadoop.hdds.scm.safemode.TestSCMSafeModeManager
-------------------------------------------------------------------------------
Tests run: 11, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.987 s <<< FAILURE! - in org.apache.hadoop.hdds.scm.safemode.TestSCMSafeModeManager
testSafeModeExitRuleWithPipelineAvailabilityCheck(org.apache.hadoop.hdds.scm.safemode.TestSCMSafeModeManager)  Time elapsed: 0.549 s  <<< FAILURE!
java.lang.AssertionError: expected:<3> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at org.apache.hadoop.hdds.scm.safemode.TestSCMSafeModeManager.testSafeModeExitRuleWithPipelineAvailabilityCheck(TestSCMSafeModeManager.java:312)
	at org.apache.hadoop.hdds.scm.safemode.TestSCMSafeModeManager.testSafeModeExitRuleWithPipelineAvailabilityCheck(TestSCMSafeModeManager.java:185)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$StatementThread.run(FailOnTimeout.java:74)

